src/main/java/dev/thatredox/chunkyaov/raytracer/DepthTracer.java

```diff
@@ -50,7 +51,8 @@ public static void register() {
     @Override
     public void trace(Scene scene, WorkerState state) {
         Ray ray = state.ray;
-        double distance = 0.0;
+        double distance = INFINITE_SKY_DISTANCE ? Double.longBitsToDouble(0x7fe0000000000000L) : 0.0;
```

Comment:
```suggestion
        double distance = INFINITE_SKY_DISTANCE ? Double.POSITIVE_INFINITY: 0.0;
```
Confusing.

Reply:
Doing it that way causes this issue when changing the camera view while rendering.

![image](https://user-images.githubusercontent.com/92183530/197741911-7d87bd49-e4a8-4e67-9710-a656f2c32ecc.png)

Reply:
Try `Double.MAX_VALUE` instead?

Reply:
That also works.

Reply:
> That also works.

What I mean is that that works properly (it does not exhibit the behavior shown in the image above).